FreshRSS/FreshRSS
# fix: highlight next/prev article while using shortcuts

Closes #5207

Fixed for themes:
- Alternative Dark
- Ansum
- Origine/Origine compact
- Pafat

I check the other themes too. They should be ok.

Changes proposed in this pull request:

- (s)CSS


How to test the feature manually:

1. go to normal view
2. each article line should have the same behavior while using next/prev shortcut like mouse hovering.


Pull request checklist:
